kairon/shared/pos/constants.py

```
@@ -26,3 +26,15 @@ class OdooPOSMenus(str, Enum):
     MENU_POS_PRODUCTS = 233
     MENU_POS_ORDERS = 231
 
+
+POS_NOTIFICATION_MESSAGES = [
+    "New POS Order Received 🎉",
+    "Order Alert 🚀 A new order just came in!",
+    "You’ve got a new order 🧾",
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)